lib/fastlane/setup/setup_ios.rb

```diff
@@ -72,6 +79,29 @@ def generate_appfile
       Helper.log.info "Created new file '#{path}'. Edit it to manage your preferred app metadata information.".green
     end
 
+    def default_run_produce
+      Helper.log.info "Running produce..."
```

Comment:
I just removed the call to this method, but i think it might be nice to have a really easy `produce` command for later on during development when someone just wants to run `produce` and not answer questions. what do you think? should i move this code somewhere  into produce to have a less-interactive version of the action?